### DISCUSSION TITLE
Is Medusa 'local' backup fully supported at the moment?

### DISCUSSION THREAD

**Discussion by carolili** [k8ssandra/k8ssandra#589] [Q&A] [answered]
Hello, could you say if the Medusa 'local' backup option fully supported at the moment?

I see there is the option to set it to local storage here: https://github.com/k8ssandra/k8ssandra/blob/main/charts/k8ssandra/values.yaml#L508

```
medusa:
  ...
  storage: "local"
  ...
```

However, there is also an ongoing issue for supporting the local storage option: https://github.com/k8ssandra/k8ssandra/issues/560

The context is that I am trying to enable Medusa with local storage, but the deployment fails, complaining that `medusa-bucket-key` doesn't exist. To my understanding the key is only for S3, and it should bot be checked. So I am wondering if the 'local' feature is fully implemented or not.

Thanks in advance!

**Comment by jsanda** [comment, marked as answer]
Hi @carolili 

What version are you running? You hit #466 which was fixed in version 1.0.1 of the k8ssandra chart. Aside from that, I wouldn't consider local storage ready at the moment. Without #560, you do not have a way a to specify the volume on which to store backups.

**Reply by carolili** [reply]
Hi @jsanda, sounds like #466 as I'm using 1.0.0, I'll make sure to use a newer version of the chart and try it again when #560 is merged and released. Thank you very much for the quick response!

**Reply by jsanda** [reply]
Sure thing 😄

@carolili I would love to hear how you are using k8ssandra. Let me know if you would be open to chatting some time.

Cheers
